specification/eventgrid/resource-manager/Microsoft.EventGrid/2017-09-15-preview/EventGrid.json

```diff
@@ -1015,6 +1015,68 @@
                     }
                 },
                 "x-ms-long-running-operation": true
+            },
+            "patch": {
+                "tags": [
+                    "Topics"
+                ],
+                "summary": "Update a topic",
+                "description": "Asynchronously updates a topic with the specified parameters.",
+                "operationId": "Topics_Update",
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "name": "resourceGroupName",
+                        "in": "path",
+                        "description": "The name of the resource group within the user's subscription.",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "topicName",
+                        "in": "path",
+                        "description": "Name of the topic",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "topicUpdateParameters",
+                        "in": "body",
+                        "description": "Topic update information",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/TopicUpdateParameters"
+                        }
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "201": {
+                        "description": "Created",
```

Comment:
nit: incorrect description

Reply:
Will update the description to "Topic update request accepted.". I just want to double-check one thing: does this description I put here have to match with anything the service returns, or is this independent?

Reply:
What the service returns must match the description since we also generate the documentation with this information, hope this answers your query

Reply:
@kalyanaj Can you fix this so we can merge the PR soon?